### PR TITLE
added learn-js

### PR DESCRIPTION
**Pull request by akulsr0**
LearnJS is a platform where beginners can learn and contribute javascript programs and questions.
I think this project could be helpful for beginners to contribute in.

### CODE REVIEW

File: _data/projects/learn-js.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+desc: A website having collection of javascript programs and questions. 
+site: https://github.com/akulsr0/learn-js
+tags:
+  - javascript
+  - programs
+  - questions
+  - algorithm
+  - data-structures
+upforgrabs:
+  name: up for grabs
```
Comment (shiftkey) [inline _data/projects/learn-js.yml:11]: This label hasn't been created in the project - should this instead be `help wanted`?

File: _data/projects/learn-js.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  link: https://github.com/akulsr0/learn-js/labels/up%20for%20grabs
```
Comment (shiftkey) [inline _data/projects/learn-js.yml:12]: And if `help wanted` is the right label, this should be the corresponding URL:

```suggestion
  link: https://github.com/akulsr0/learn-js/labels/help%20wanted
```

**Review by shiftkey — COMMENTED** [review]
Thanks for the submission @akulsr0! Just a couple of suggestions to wrap this up...

File: _data/projects/learn-js.yml (hunk 2 above)
Comment (akulsr0) [reply _data/projects/learn-js.yml:12]: Created a new PR!

Comment (shiftkey) [reply _data/projects/learn-js.yml:12]: @akulsr0 for next time you're working on a PR, you can continue to push commits to your branch after opening the PR and it'll synchronize the changes - no need to start fresh, and makes reviewing easier.

**Comment by shiftkey** [comment]
Moving over to #2123

**Pull request closed without merging**